src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+
+### Video CSS: `display`
+
+To begin, let's look at how the video will appear on the page by looking at the CSS `display` property for the video. What we find is that approximately half of all videos use a display value of `block`—placing the video on its own line and allowing for height and width values to be set for the video. The `inline-block` value also  allows height and width to be specified—for a total of two third's of all videos.
+
+The  `display: none` declaration hides the video from the viewer. One in five videos on the web is hidden behind this display value. From a data usage perspective, this is less than optimal, as the video is still downloaded by the browser.
+
+{{ figure_markup(
+  image="video-css-display-percentages.png",
+  caption="Video CSS display percentages.",
+  description="A bar chart showing the percentage of video elements with various values of the CSS display property. `block` accounts for 51% of elements on desktop, and 45% on mobile. `inline` accounts for 15% of elements on desktop, and 17% on mobile. `inline-block` accounts for 15% of elements on desktop, and 16% on mobile. Finally, `none` accounts for 18% on desktop and 21% on mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1841222549&format=interactive",
+  sheets_gid="428232209",
+  sql_file="video_styles_mobile.sql"
+  )
+}}
+
+### Video attributes
+
+The video HTML5 tag has a number of attributes that can be used to define how the video player will appear to end users.
+
+Let's look at the most common attributes and how they are used inside the `<video>` tag:
+
+{{ figure_markup(
+  image="video-element-attributes.png",
+  caption="Video element attributes.",
+  description="A bar chart showing the number of times various attributes on the HTML video element were found, on both desktop and mobile. With the exception of `controls`, which is slightly more common on mobile, every attribute is more common on desktop by 5-30%. The tallest pair of bars is `preload`; `buffered`s are the shortest. None of the bars are labelled, but their desktop numbers are approximately: `autoplay`: 100,000; `buffered`: 0; `controls`: 45,000; `controllist`: 1,000; `crossorigin`: 30,000; `height`: 3,000; `loop`: 50,000; `muted`: 55,000; `playsinline`: 52,000; `poster`: 24,000; `preload`: 115,000; `src`: 90,000; width: 60,000.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1099468813&format=interactive",
+  sheets_gid="1827909463",
+  sql_file="video_attribute_count_mobile.sql"
+  )
+}}
```

Comment:
Can we change these to percentage of videso? The raw numbers are kind of meaningless without context and less easily compared year on year.

Reply:
I think this is a good idea. Thoughts, @dougsillars?

Reply:
Changed to percentages.